unittests/Feature/FeatureModelTransaction.cpp

```diff
@@ -904,4 +904,128 @@ TEST_F(FeatureModelTransactionTest, removeFeatureFromModel) {
   EXPECT_FALSE(FM->getFeature("ab"));
 }
 
+TEST_F(FeatureModelTransactionTest, removeFeaturesFromModel) {
+  size_t FMSizeBefore = FM->size();
+
+  // Prepare Model with several Features
+  vara::feature::addFeature(*FM, std::make_unique<BinaryFeature>("ab"),
+                            FM->getFeature("a"));
+  vara::feature::addFeature(*FM, std::make_unique<BinaryFeature>("ac"),
+                            FM->getFeature("a"));
+  vara::feature::addFeature(*FM, std::make_unique<BinaryFeature>("ad"),
+                            FM->getFeature("ab"));
+  vara::feature::addFeature(*FM, std::make_unique<BinaryFeature>("ae"),
+                            FM->getFeature("ad"));
```

Comment:
Yes, but since it does a commit in the very end, I cannot use it to create a tree-like hierarchy of my features. It would work out if all Features are just children of a.

Reply:
Oh, hmm then maybe we need to adapt the impl of addFeatures. My inital thoughts were that this kind of scenario should be possible. So, maybe we need add them all an after wards commit. The only thing we need to make sure there is that parents are added before children because otherwise the add modification would not work.

Reply:
Can you please create an issue so we don't forget this in the future and like it with a comment to the corresponding locations. The format ist `TODO (se-sic/VaRA#???): short description`

Reply:
Done